protege/pages/contributors.js

```diff
@@ -58,4 +142,52 @@ const Contributors = () => {
   )
 }
 
-export default Contributors
+Contributors.propTypes = {
+  contributors: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
This is the full list of properties for a contributor, but maybe overfill for this component. I could shrink it down to what we're only consuming.

Reply:
Yeah, seems like we could remove a lot of these since the component only uses avatar, name, twitter, github, blog and number of commits.